setup/pkg/fieldlabs/environment_create.go

```diff
@@ -210,24 +211,24 @@ func (e *EnvironmentManager) mergeWriteTFInstancesJSON(labStatuses []Lab) error
 				InstallScript: fmt.Sprintf(`
 #!/bin/bash 
 # add new user
-sudo useradd -s /bin/bash -d /home/kots -m -p safWNrcAGYqm2 -G sudo kots
-sudo groups kots
-echo 'kots ALL=(ALL)        NOPASSWD: ALL' | sudo EDITOR='tee -a' visudo
+sudo useradd -s /bin/bash -d /home/%[1]v -m -p safWNrcAGYqm2 -G sudo %[1]v
+sudo groups %[1]v
+echo '%[1]v ALL=(ALL)        NOPASSWD: ALL' | sudo EDITOR='tee -a' visudo
 # update ssh to allow password login
 sudo sed -i 's/no/yes/g' /etc/ssh/sshd_config
 sudo service ssh restart
-# add kots to google-sudoers
-sudo usermod -aG google-sudoers,kots kots
+# add %[1]v to google-sudoers
+sudo usermod -aG google-sudoers,%[1]v %[1]v
 # user must change password on first login
-sudo chage --lastday 0 kots
+sudo chage --lastday 0 %[1]v
 
 set -euo pipefail
 
 mkdir -p ~/.ssh
 cat <<EOF >>~/.ssh/authorized_keys
 %s
```

Comment:
Will it still pickup this `%s`? Just not sure, and my go knowledge doesn't reach that far yet.

Reply:
The `%s` will pick up a blank and pass into the authorized_keys. If we add an SSH key it will populate it.